foe-ios/Models/Sighting.swift

```diff
@@ -67,19 +67,15 @@ class Sighting {
   }
 
   func getWeather() -> String {
-    return self.weather!
+    return (self.weather != nil) ? self.weather! : "";
```

Comment:
(same as above) is weather ever nil?

Reply:
it's nil when it's not yet assigned in Capture flow